src/modules/onboarding/components/connect-to-wpcom.tsx

```diff
@@ -16,11 +17,30 @@ export function OnboardingConnectToWpcom( { onSkip }: { onSkip: () => void } ) {
 
 	return (
 		<div className="h-full flex flex-col">
-			<div className="flex flex-col items-center gap-6 my-auto">
-				<Heading className="text-[32px] text-gray-900 text-center px-10" weight={ 500 }>
-					{ __( 'Connect to your WordPress.com account' ) }
+			<div className="flex flex-col gap-4 my-auto text-pretty">
+				<Heading className="a8c-subtitle text-xl">
+					{ __( 'Connect your WordPress.com account' ) }
 				</Heading>
 
+				<div className="text-a8c-gray-70 a8c-body">
+					{ __(
+						'Log in with WordPress.com to unlock the full power of WordPress Studio. By signing in, you get access to features that help you build faster, test safely, and work seamlessly across environments:'
+					) }
+				</div>
+
+				<div>
+					{ [
+						__( 'Share preview sites with team members and clients.' ),
```

Comment:
```suggestion
						__( 'Share preview sites with clients and team members.' ),
```

Just wanted to put "clients" first here as I think this is what most people care about in the first place